README.md

```diff
@@ -53,6 +53,36 @@ and Erlang/OTP.
 
 **Note** *: prior to 23, Windows builds are only available for minor versions, e.g. 21.0, 21.3, 22.0, etc.
 
+### Self-hosted runners
+
+We try to guess the self-hosted runner _container_ equivalent (to download assets used in the action)
+by making direct calls to the operating system, but this might fail, in which case we
+warn with an error message, and you should set environment variable `ImageOS` to one of the following:
+
+| ImageOS  | Operating system
+|-         |-
+| ubuntu16 | ubuntu-16.04
+| ubuntu18 | ubuntu-18.04
+| ubuntu20 | ubuntu-20.04
+| win16    | windows-2016
+| win19    | windows-2019
+
+as per the following example:
+
+```yaml
+...
+
+jobs:
+  test:
+    runs-on: self-hosted
+    env:
+      ImageOS: ubuntu20 # equivalent to runs-on ubuntu-20.04
```

Comment:
This is our safety net in case the "convoluted" guess doesn't work as expected (it's tested in all versions the action supports, as of now)

Reply:
This comment no longer applies, since there's no guesswork